exporter/datadogexporter/metrics_exporter_test.go

```diff
@@ -473,21 +473,20 @@ func Test_metricsExporter_PushMetricsData_Zorkian(t *testing.T) {
 						"host":   "test-host",
 						"tags":   []interface{}{"lower_bound:0", "upper_bound:inf", "env:dev"},
 					},
+					map[string]interface{}{
+						"metric": "otel.system.disk.in_use",
```

Comment:
Note to reviewer: I'm not sure why this wasn't prepended with `otel.*` before. It seems like an error.